apps/daimo-mobile/src/view/screen/history/shared.ts

```diff
@@ -18,15 +18,18 @@ export function getSynthesizedMemo(
   if (op.type !== "transfer") return null;
   if (op.memo) return op.memo;
   if (op.preSwapTransfer) {
-    if (op.preSwapTransfer.coin.token === "ETH") return null;
+    if (op.preSwapTransfer.coin.token === "ETH") {
+      return `ETH → ${coinName}`;
+    }
```

Comment:
cc @nalinbhardwaj for post review
this is the main change, found it during release testing.